### PR TITLE
Preliminary MAX78002 support; interrupt driven operation

### PR DESCRIPTION
**Pull request by rotx-maxim**

### CODE REVIEW

**Review by aniktash — APPROVED** [review]
you may want to update copyright on Makefile.ARM_PROG    (8 files)

**Review by vicloginov — APPROVED** [review]

File: commandline.py [hunk 1]
```diff
@@ -57,6 +67,8 @@ def get_parser():
                        help="show parsed checkpoint data")
     group.add_argument('--prefix', metavar='S', required=True,
                        help="set test name prefix")
+    group.add_argument('--debugwait', type=int, default=2, metavar='N',
+                       help="set the delay in seconds before calling __WFI() (default: 1)")
```
Comment (vicloginov) [inline commandline.py:71]: Default=2

File: device-ai87/test_softmax.c [hunk 2]
```diff
@@ -0,0 +1,226 @@
+/**
+ * @file    test_softmax.c
+ * @brief   Test software Softmax
+ * @details This test evaluates the Softmax functions.
+ */
+
+/*******************************************************************************
+ * Copyright (C) 2016-2020 Maxim Integrated Products, Inc., All Rights Reserved.
```
Comment (vicloginov) [inline device-ai87/test_softmax.c:8]: Remove year

File: rtldev/ai85-biasmem/Makefile.ARM_PROG [hunk 3]
```diff
@@ -0,0 +1,130 @@
+################################################################################
+ # Copyright (C) 2016 Maxim Integrated Products, Inc., All Rights Reserved.
```
Comment (vicloginov) [inline rtldev/ai85-biasmem/Makefile.ARM_PROG]: Remove year

File: rtldev/ai85-datamem/Makefile.ARM_PROG [hunk 4]
```diff
@@ -0,0 +1,130 @@
+################################################################################
+ # Copyright (C) 2016 Maxim Integrated Products, Inc., All Rights Reserved.
```
Comment (vicloginov) [inline rtldev/ai85-datamem/Makefile.ARM_PROG:2]: Remove year

File: rtldev/ai85-kernelmem-mexpress/Makefile.ARM_PROG [hunk 5]
```diff
@@ -0,0 +1,130 @@
+################################################################################
+ # Copyright (C) 2016 Maxim Integrated Products, Inc., All Rights Reserved.
```
Comment (vicloginov) [inline rtldev/ai85-kernelmem-mexpress/Makefile.ARM_PROG:2]: Remove year

**Review by vicloginov — COMMENTED** [review]
Makefile.ARM_PROG has year in header

**Review by vicloginov — APPROVED** [review]

File: rv.py [hunk 6]
```diff
@@ -1,10 +1,8 @@
 ###################################################################################################
-# Copyright (C) 2018-2020 Maxim Integrated Products, Inc. All Rights Reserved.
```
Comment (vicloginov) [inline rv.py:2]: Remove year

**Review by vicloginov — APPROVED** [review]

**Pull request merged**